net/udprelay/server.go

```diff
@@ -81,6 +81,63 @@ func newPairOfDiscoPubKeys(discoA, discoB key.DiscoPublic) pairOfDiscoPubKeys {
 	return pair
 }
 
+// ServerEndpointLamportID implements encoding.TextMarshaler and
+// encoding.TextUnmarshaler interfaces for consistent cross-platform
+// serialization behaviors.
+type ServerEndpointLamportID uint64
```

Comment:
How are these IDs generated? For IDs generated by control, we deliberately limit them to the number space of [0..2^53] so that the natural JSON number representation wouldn't ever have precision issues.

Reply:
LamportID increments every `Server.AllocateEndpoint()`, so in theory it shouldn't exceed 1<<53-1 within our lifetimes.

I'm going to simplify this PR to just time.Duration as a JSON string once we have the generic type for that in the tstime package.